Casks/z/zenmap.rb

```diff
@@ -8,8 +8,7 @@
   homepage "https://nmap.org/zenmap/"
 
   livecheck do
-    url "https://nmap.org/dist/?C=M;O=D"
-    regex(/href=.*?nmap[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
+    formula "nmap"
```

Comment:
```suggestion
    url "https://nmap.org/download.html"
    regex(/href=.*?nmap[._-]v?(\d+(?:\.\d+)+)\.dmg/i)
```

We generally only use a formula/cask reference when the `stable`/cask URL are the same. In this case, the `nmap` `livecheck` block matches the version from the tarball link on the download page but the cask is using the dmg file, so it's not appropriate to use `formula "nmap"` here.